src/components/label/label.tsx

```diff
@@ -0,0 +1,87 @@
+import { getIconName } from '../icon/get-icon-props';
+import { Component, Prop, h } from '@stencil/core';
+import { Label, LabelValue } from './label.types';
+import { Icon } from '../../interface';
+
+/**
+ * This components displays a different label depending on the current given
+ * value. A label can consist of a text and an icon. If no matching label is
+ * found, the default label will be displayed.
```

Comment:
To me this description is unclear:

> If no matching label is found, the default label will be displayed.

Reply:
4acef8ba1e5df8df25d68bb4b0b8476246d42dcf